From f1791f2bdbe3ee4da29965314fc5879eb066d24f Mon Sep 17 00:00:00 2001
From: Ran Wang <ran.wang_1@nxp.com>
Date: Tue, 11 Dec 2018 15:57:06 +0800
Subject: [PATCH 400/706] arm: dts: ls1028a: Enable usb3-lpm-capable for usb3
 node

Enable USB3 HW LPM feature for ls1028a and active patch for
snps erratum A-010131. It will disable U1/U2 temperary when
initiate U3 request.

Signed-off-by: Ran Wang <ran.wang_1@nxp.com>
(cherry picked from commit 766a2015141518fbd4fe56c4714e8b28fd53aaf9)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 913a073da9d0..754d30f5baee 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -369,6 +369,8 @@
 			snps,dis_rxdet_inp3_quirk;
 			snps,quirk-frame-length-adjustment = <0x20>;
 			snps,incr-burst-type-adjustment = <1>, <4>, <8>, <16>;
+			usb3-lpm-capable;
+			snps,dis-u1u2-when-u3-quirk;
 		};
 
 		usb1:usb3@3110000 {
@@ -379,6 +381,8 @@
 			snps,dis_rxdet_inp3_quirk;
 			snps,quirk-frame-length-adjustment = <0x20>;
 			snps,incr-burst-type-adjustment = <1>, <4>, <8>, <16>;
+			usb3-lpm-capable;
+			snps,dis-u1u2-when-u3-quirk;
 		};
 
 		i2c0: i2c@2000000 {
-- 
2.17.1

